CSCore/CSCore.csproj

```diff
@@ -619,10 +619,12 @@
   <ItemGroup>
     <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
CSCore.csproj still contains SoundTouch.dll. Also the Libs/SoundTouch.dll file still exists within CSCore.